# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../7zip/cross.patch
# Copyright (C) 2007 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- p7zip_4.48/makefile.linux_x86_ppc_alpha.vanilla	2007-07-03 16:05:02.000000000 +0000
+++ p7zip_4.48/makefile.linux_x86_ppc_alpha	2007-07-03 16:05:41.000000000 +0000
@@ -12,6 +12,11 @@
 
 CXX=g++ $(ALLFLAGS)
 CC=gcc $(ALLFLAGS)
+CXX=g++
+CC=gcc
+override CXX += $(ALLFLAGS)
+override CC += $(ALLFLAGS)
+
 CC_SHARED=-fPIC
 LINK_SHARED=-fPIC -shared
 
